binderhub/templates/index.html

```diff
@@ -111,6 +97,40 @@ <h4 id="form-header">Build and launch a repository</h4>
             <div id="log"></div>
           </div>
         </div>
+
+        <!--url section-->
+        <div class="url">
+            <div class="bluedropdown">
+              <label>Copy and share this URL:</label>
+            </div>
+            <div class="url-row">
+              <pre id="basic-url-snippet"><a href="#" id="badge-url" title="show url snippet">Begin typing to see your URL!</a></pre>
+            </div>
+        </div>
+
+        <div class="badges">
+            <div class="bluedropdown" id="toggle-badge-snippet">
+              <label>Click, then paste into your README to get badge: <img id="badge" src="/static/images/badge.svg"></label>
+              <a id="badge-link"></a>
+              <a href="#" title="show badge snippets"><span class="caret"></span></a>
+
+              </button>
+            </div>
+            <div id="badge-snippets" class="hidden">
+              <!--Markdown section-->
+              <div  class="badge-snippet-row">
+                 <pre id="markdown-badge-snippet">Fill in the fields to see the markdown badge snippet</pre>
+                 <img class="icon" src="/static/images/markdown-icon.svg">
```

Comment:
like here

Reply:
can you clarify what you mean here? I'll be around gitter if you wanna chat there and get this done!